File: newcoder_0/1019/python/main.py
import sys

_in = sys.stdin.next()
_out = sys.stdin.next()
i = 0
show = [0] * 128
for c in _in:
    if i < len(_out) and _out[i] == c:
        i += 1
    else:
        cur = ord(c) - 32 if c >= 'a' and c <= 'z' else ord(c)
        if show[cur] == 0:
            show[cur] = 1
            sys.stdout.write("%c" % cur)
sys.stdout.write("\n")

File: newcoder_0/1019/c/main.c
#include <stdio.h>
#include <string.h>

int main() {
  char in[81], out[81], show[60], cur;
  memset(show, 0, 92);
  scanf("%s\n%s\n", in, out);
  for(char *p = in, *q = out; *p != 0; ++p) {
    if(*p == *q) {
      ++q;
    } else {
      if(*p >= 'a' && *p <= 'z') {
        cur = *p - 32;
      } else {
        cur = *p;
      }
      if(show[cur-' '] == 0) {
        show[cur-' '] = 1;
        printf("%c", cur);
      }
    }
  }
  printf("\n");
  return 0;
}

